### PR TITLE
corrected navbar for new home templete

### PR DESCRIPTION
**Pull request by agamble33**
updated with new home templete specifications

### CODE REVIEW

File: chucks-pick3-web/assets/css/style-sheet.css [hunk 1]
```diff
@@ -272,12 +272,17 @@ link-container {
 
 .how-to-container {
     display: block;
-    background-color: #a9c0ee;
-    width: 33.33%;
-    height: 300px;
-    padding: 0px;
-    margin: auto;
-    position: center;
+    background-image: url("../img/web icons-01.png");
```
Comment (pouncilt) [inline chucks-pick3-web/assets/css/style-sheet.css]: Where is this image located at?  Do we really need this image?

Comment (pouncilt) [inline chucks-pick3-web/assets/css/style-sheet.css]: Any reason why we are updating the .how-to-container rule?

Comment (agamble33) [reply chucks-pick3-web/assets/css/style-sheet.css]: I updated and removed bullets and made width equal for each div

Comment (agamble33) [reply chucks-pick3-web/assets/css/style-sheet.css]: corrected

**Comment by pouncilt** [comment]
work already done.

**Pull request closed without merging**